components/blog.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+type BlogType = {
+  url: string;
+  title: string;
+  profilePic: string;
+  date: string;
+  name: string;
+};
+
+type BlogProps = {
+  blog: BlogType;
+  index: number;
+};
+
+export const Blog: React.FC<BlogProps> = ({ blog, index }) => {
+  return (
+    <div id="blog-main" style={{ animationDelay: `${200 * index}ms` }}>
```

Comment:
could this animationDelay be in the `<style jsx>` section? (not sure if this is required for some reason) but no need to change this, it's fine to have it here as it is variable